core/examples/json/JobList.json

```diff
@@ -1,61 +1,83 @@
-[
-	{
-		"jobID": "8ca109b4-3b86-4a9c-a284-a6d50f91019e",
-		"status": "running",
-		"message": "Perform step 1/2",
-		"progress": 50,
-		"links": [
-			{
-				"href": "http://processing.example.org/oapi-p/jobs/8ca109b4-3b86-4a9c-a284-a6d50f91019e",
-				"rel": "status",
-				"type": "application/json",
-				"hreflang": "en",
-				"title": "Job status"
-			}
-		]
-	},
-	{
-	    "jobID": "0cf773a5-282a-4e23-96cc-f5dab18123e5",
-	    "status": "successful",
-	    "message": "EchoProcess job finished successful",
-	    "progress": 100,
-	    "links": [
-		{
-		    "href": "http://processing.example.org/oapi-p/jobs/0cf773a5-282a-4e23-96cc-f5dab18123e5",
-		    "rel": "status",
-		    "type": "application/json",
-		    "hreflang": "en",
-		    "title": "Job status"
-		},
-		{
-		    "href": "http://processing.example.org/oapi-p/jobs/0cf773a5-282a-4e23-96cc-f5dab18123e5/results",
-		    "rel": "results",
-		    "type": "application/json",
-		    "hreflang": "en",
-		    "title": "Job result"
-		}
-	    ]
-	},
-	{
-	    "jobID": "63aadd9c-c0e5-4a7f-80f0-228dbb158f09",
-	    "status": "failed",
-	    "message": "EchoProcess job failed",
-	    "progress": 100,
-	    "links": [
-		{
-		    "href": "http://processing.example.org/oapi-p/jobs/63aadd9c-c0e5-4a7f-80f0-228dbb158f09",
-		    "rel": "status",
-		    "type": "application/json",
-		    "hreflang": "en",
-		    "title": "Job status"
-		},
-		{
-		    "href": "http://processing.example.org/oapi-p/jobs/63aadd9c-c0e5-4a7f-80f0-228dbb158f09/results",
-		    "rel": "exceptions",
-		    "type": "application/json",
-		    "hreflang": "en",
-		    "title": "Job exception"
-		}
-	    ]
-	}
-]
+{
+  "jobList": [
```

Comment:
Since the API request is `GET /jobs`, shouldn't the name be `"jobs"` instead of `"jobList"` ?

Reply:
@fmigneault already changed in the PR. "processList" -> "processes", "jobList" -> "jobs".